config/main.default.coffee

```diff
@@ -41,7 +41,6 @@ Configuration = (options = {}) ->
   options.sendEventsToSegment = yes
   options.scheme = 'http'
   options.suppressLogs = no
-  options.paymentBlockDuration = 2 * 60 * 1000 # 2 minutes
```

Comment:
👍  

I was using this for blocking people who tried to enter a failing card more than 3 times. But since we have a auth system in backend now, i don't think this will be necessary.